client/components/Assignments/AssignmentHistory.tsx

```diff
@@ -108,7 +112,9 @@ class AssignmentHistoryComponent extends React.Component {
     }
 
     render() {
-        const getHistoryActionUserName = (userId) => getItemInArrayById(this.props.users, userId).display_name;
+        const getHistoryActionUserName = (userId) =>
```

Comment:
You could use Typescript's `Optional Chaining` and `Nullish Coalescing` operators so that `getItemInArrayById` isn't called twice.
i.e.
```typescript
getItemInArrayById(this.props.users, userId)?.display_name ?? ''
```

Reply:
Very nice, will give it a go!